providers/weaviate/tests/unit/weaviate/hooks/test_weaviate.py

```diff
@@ -624,6 +627,8 @@ def test_delete_by_property_retry(get_conn, weaviate_hook):
     assert mock_collection.data.delete_many.call_count == len(side_effect)
 
 
+# TODO: fix me for PY313 the tests expect httpx objects rather than requests
```

Comment:
Is this TODO to be fixed before merge or shall we add a Github issue for this not to forget?